docs/web_advanced.rst

```diff
@@ -896,20 +896,26 @@ background tasks could be registered as an :attr:`Application.on_startup`
 signal handler or :attr:`Application.cleanup_ctx` as shown in the example
 below::
 
-
-  async def listen_to_redis(app):
-      try:
-          sub = await aioredis.create_redis(('localhost', 6379))
-          ch, *_ = await sub.subscribe('news')
-          async for msg in ch.iter(encoding='utf-8'):
-              # Forward message to all connected websockets:
-              for ws in app[websockets]:
-                  ws.send_str('{}: {}'.format(ch.name, msg))
-      except asyncio.CancelledError:
-          pass
-      finally:
-          await sub.unsubscribe(ch.name)
-          await sub.quit()
+  import redis.asyncio as redis
+  from aiohttp import web
+
+
```

Comment:
```suggestion
```

Reply:
Actually, I'd suggest removing the imports as well. This is meant to be a concise example of how to do something, not a fully working demo.